src/app/_types/markdown.d.ts

```diff
@@ -1,3 +1,9 @@
+declare module '../security/maturity-model/content/*.md' {
```

Comment:
```suggestion
declare module '../content/*.md' {
```
This seems to resolve the module import

Reply:
Now, if we only knew why. 😂